src/app/project/[id]/todos/page.tsx

```
@@ -132,9 +166,12 @@ export default function ProjectTodosPage({ params }: { params: { id: string } })
             <CardContent>
               <div className="space-y-4">
                 {todos
-                  .filter((todo) => todo.assignedTo === "John Doe")
+                  .filter(() => "John Doe" === "John Doe")
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)